docs/src/getting_started.md

```diff
@@ -18,6 +18,11 @@ Then start the simulation by executing
 julia> trixi_include(joinpath(examples_dir(), "fluid", "hydrostatic_water_column_2d.jl"))
 ```
 
+The easiest way to quickly visualize the result is to use Plots.jl:
+```jldoctest getting_started; filter = r".*"s, setup=:(trixi_include(@__MODULE__, joinpath(examples_dir(), "fluid", "hydrostatic_water_column_2d.jl")))
```

Comment:
I wish I could avoid `filter` to actually test that a plot comes out, but `output = false` doesn't work here, so it would show `Plot{Plots.GRBackend() n=1800}` in the docs, which is ugly.